node_modules/col-core/lib/db.js

```diff
@@ -638,6 +639,10 @@ var setUpModel = function(sequelize) {
   // A whiteboard can have multiple whiteboard elements
   Whiteboard.hasMany(WhiteboardElement, {'as': 'WhiteboardElements'});
 
+  // An asset can be used as an element in multiple whiteboards. Note that this relation differs from the
+  // AssetWhiteboardElement relation, which describes elements in whiteboards that have been exported as assets
+  Asset.hasMany(WhiteboardElement, {'as': 'whiteboard_usages'});
```

Comment:
That is probably correct. Defining these relationships always confuses me, so I just check the database to see if anything changed.